packages/sdk/src/utils/token-list-utils.ts

```diff
@@ -0,0 +1,50 @@
+import { fetchJson } from 'ethers/lib/utils'
+
+import { OPTIMISM_TOKENLIST_URI } from './misc-constants'
+
+interface Options {
+  uri?: string
+  chainId?: number
+  filterTerm?: string
+}
+
+export interface TokenList {
+  name: string
+  logoURI: string
+  tokens: TokenListItem[]
+}
+
+export interface TokenListItem {
+  chainId: number
+  address: string
+  name: string
+  symbol: string
+  decimals: number
+  logoURI?: string
+  extensions?: {
+    optimismBridgeAddress?: string
+    ens?: string
+  }
+}
+
+export const fetchTokenList = async ({
+  uri = OPTIMISM_TOKENLIST_URI,
+  chainId,
+  filterTerm,
+}: Options = {}): Promise<TokenList> => {
```

Comment:
Just to make sure I am not missing something, what is the reason for having those options if are not needed/used in this PR?

Reply:
They are going to be used in the gateway that's where this came from.  I'm exporting this as a common util for anybody to use 
![image](https://user-images.githubusercontent.com/35039927/189973643-74876736-25a0-4f75-b721-3f7e56162415.png)